knowledge-base/troubleshoot-report-server.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Troubleshooting Report Server
+description: Diagnose problems with the Telerik Report Server application
+type: troubleshooting
+page_title: Troubleshoot Report Server Manager
+slug: troubleshoot-report-server
+position: 
+tags: 
+ticketid: 1400116
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Report Server</td>
+	</tr>
+</table>
+
+
+## Description
+Sometimes it is possible to experience problems with the Report Server the reasons for which are not apparent. For example, restricted permission for accessing the File Storage, Internal Server Errors due to unaccessible Storage or missing/corrupted report resources (e.g. invalid image), etc.
+
+## Solution
+We recommend using the following basic approaches for troubleshooting :
+  
+1. Check whether there are any errors in the browser console.  
+
+2. Use [Fiddler ](https://www.telerik.com/download/fiddler) to see the exact requests from the Report Manager Viewer to the server, and the corresponding responses - they may indicate what has failed.  
```

Comment:
What is a Report Manager Viewer ?

Reply:
changed to "...requests from the Viewer of the Report Server Manager to the Report Server REST Service..."